actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -252,13 +252,17 @@ def permit!
     #
     #   ActionController::Parameters.new(person: {}).require(:person)
     #   # => ActionController::ParameterMissing: param is missing or the value is empty: person
+    #
+    #   ActionController::Parameters.new(first_name: 'Gaurish', title: nil).require([:first_name, :title])
+    #   # => ActionController::ParameterMissing: param is missing or the value is empty: title
+    #
+    #   params = ActionController::Parameters.new(first_name: 'Gaurish', title: Mjallo)
+    #   first_name, title = params.require([:first_name, :title])
+    #
     def require(key)
+      return keys.map{|k| require(k)} if key.is_a?(Array)
       value = self[key]
-      if value.present? || value == false
-        value
-      else
-        raise ParameterMissing.new(key)
-      end
+      (value.present? || value == false) ? value : raise(ParameterMissing.new(key))
```

Comment:
My point was that these lines shouldn't be changed at all. Can you roll back to how it was before your change?